apis/livepeer/livepeer.go

```diff
@@ -383,36 +376,27 @@ func (lapi *API) DeleteStream(id string) error {
 
 // CreateStreamEx creates stream with specified name and profiles
 func (lapi *API) CreateStreamEx(name string, record bool, presets []string, profiles ...Profile) (*CreateStreamResp, error) {
-	return lapi.CreateStreamEx2(name, record, "", presets, profiles...)
+	return lapi.CreateStream(CreateStreamReq{Name: name, Record: record, Presets: presets, Profiles: profiles})
 }
 
-// CreateStreamEx creates stream with specified name and profiles
-func (lapi *API) CreateStreamEx2(name string, record bool, parentID string, presets []string, profiles ...Profile) (*CreateStreamResp, error) {
```

Comment:
Looks like we are getting rid of these, great they were pretty confusing - do we need to check if this will break builds elsewhere?

Reply:
I wouldn't be much worried about it since the build will only reaaally break when they do the `go get -u` on this package. What I'm a little more worried is that a little later, for VOD, I created this separate repository to hold the API client instead of having everyone depending on this internal package: https://github.com/livepeer/go-api-client

So these changes should be upstreamed there and we should eventually, hopefully soon, migrate this repository to use that lib instead of having it's own implementation of a Livepeer API client. That other repository started exactly from this code tho, so it should not be very hard to migrate anyway.

Reply:
Right makes sense, will keep an eye out for using `go-api-client`